sql/core/src/test/scala/org/apache/spark/sql/connector/SimpleWritableDataSource.scala

```diff
@@ -222,7 +217,7 @@ class CSVDataWriter(fs: FileSystem, file: Path) extends DataWriter[InternalRow]
   private val out = fs.create(file)
 
   override def write(record: InternalRow): Unit = {
-    out.writeBytes(s"${record.getLong(0)},${record.getLong(1)}\n")
+    out.writeBytes(s"${record.getInt(0)},${record.getInt(1)}\n")
```

Comment:
This change just matches the TestingV2Source.schema?

Reply:
Yea